templates/openshift-prune-cronjob.yml.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: build-pruner
+spec:
+  schedule: "{{ osbs_prune_schedule }}"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          volumes:
+          - name: token
+            secret:
+              defaultMode: 420
+              secretName: {{ osbs_prune_secret }}
+          containers:
+          - name: build-pruner
+            image: {{ osbs_prune_image }}
```

Comment:
What does this image look like? Will it be maintained by OpenShift via a BuildConfig?

Reply:
> What does this image look like? Will it be maintained by OpenShift via a BuildConfig?

The most easy way to that is to include necessary utils and scripts in the buildroot. For other instances it could be a separate image with tools only